lib/Pulse.Lib.AVLTree.fsti

```diff
@@ -0,0 +1,110 @@
+//Pulse AVL tree interface. Inspired from Steel. The FStar spec file is adopted from Steel
+//----------------------------------------------------------------------------------------------------------
+module Pulse.Lib.AVLTree
+open Pulse.Lib.Pervasives
+module S = Pulse.Lib.Stick.Util
+module FA = Pulse.Lib.Forall.Util
+open FStar.List.Tot
+
+module T = Spec.AVLTree
+module G = FStar.Ghost
+
+noeq
```

Comment:
As far as I can see you can hide node, node_ptr from the interface and make the implementation of tree_t hidden from the interface.

Reply:
comment addressed.